src/phpDebug.ts

```diff
@@ -197,8 +197,9 @@ class PhpDebugSession extends vscode.DebugSession {
 
     /** converts a server-side XDebug file URI to a local path for VS Code with respect to source root settings */
     protected convertDebuggerPathToClient(fileUri: string): string {
-        // convert the file URI to a path
-        const serverPath = decodeURI(url.parse(fileUri).pathname.substr(1));
+        // convert the file URI to a path. Don't remove starting slash on unix platforms.
+        let n:number = (process.platform === 'win32') ? 1 : 0;
```

Comment:
Could you add a space between `n:` and `number` please? I will add TSLint soon :)